docs/community.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Community
 type: docs
-weight: 5
+weight: 6
```

Comment:
I thought it made sense to put this after the Frameworks Repository in the list.